language_service/src/protocol.rs

```diff
@@ -89,6 +89,12 @@ pub struct Hover {
     pub span: Range,
 }
 
+#[derive(Debug, PartialEq)]
+pub struct TextEdit {
```

Comment:
Hmm, ideall this would be the same data type as the one used by `additional_text_edits` in `CompletionItem`, because they're defined the same type in the vscode/lsp api. Could you refactor `CompletionItem` to match?

Reply:
It is now using the TextEdit defined in the LS protocol.rs file. Thanks!